worlds/LauncherComponents.py

```diff
@@ -78,9 +78,9 @@ def __call__(self, path: str) -> bool:
         return False
 
 
-def launch_textclient():
+def launch_textclient(*args):
```

Comment:
In Launcher.py `main()`, do you think we could change

`run_component(args["component"], *args["args"])`

to

```
try:
    run_component(args["component"], *args["args"])
except TypeError as e:
    warn(f"Component {args["component"]} does not know how to take args {args["args"]}."}
    run_component(args["component"])
```

Reply:
Idk if `except TypeError as e` is the best way to check for "this function does not take this many arguments", if there is a better way that would be preferable ofc

Reply:
`if len(inspect.signature(x).parameters)` looks to be the most direct way if you really want to check there are any args defined
are you just asking for it so this is less of a breaking change (when args are used, which isn't very often) or do you actually think adding some code to strip args if unneeded is important to the ecosystem?

Reply:
Yeah idk this is probably not that important you're right